components/cors-mgt/org.wso2.carbon.identity.cors.mgt.core/src/main/java/org/wso2/carbon/identity/cors/mgt/core/constant/ErrorMessages.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.cors.mgt.core.constant;
+
+/**
+ * ErrorMessages enum holds the error codes and messages.
+ */
+public enum ErrorMessages {
+
+    ERROR_CODE_CORS_CRUD("CMA-65001", "Error in %s while %s the CORS Origins."),
```

Comment:
Shall we chage the code to CMS - CORS Management Service. 
That's how you have named the OSGI service right